src/terms/ewald.jl

```diff
@@ -82,56 +82,45 @@ function energy_ewald(lattice, recip_lattice, charges, positions; η=nothing, fo
     max_exp_arg = -log(eps(T)) + 5  # add some wiggle room
     max_erfc_arg = sqrt(max_exp_arg)  # erfc(x) ~= exp(-x^2)/(sqrt(π)x) for large x
 
+    # Precomputing summation bounds from cutoffs.
+    # Let A be the lattice matrix, then if ||Ax|| <= R, 
+    # then xi = <ei, A^-1 Ax> = <A^-T ei, Ax> <= ||A^-T ei|| R.
+    #
+    # Reciprocal space:  ||A^-1 G|| / 2η ≤ max_erfc_arg
+    # Real space: ||A(ti - tj - R)|| * η ≤ max_erfc_arg
```

Comment:
with A the real-space lattice

Reply:
(i know, but the reader of the code does not at that point :smile:)